scss/detail.scss

```diff
@@ -1,362 +1,321 @@
       }
-
-      .logout{
+      
+      .badge {
         position: absolute;
-        top: 26px;
-        right: 246px;
-        bottom: 26px;
-        left: 1057px;
-        color: $black;
-        font-size: 18px;
-        line-height: 21px;
+        display: flex;
+        height: 119px;
+        width: 95px;
+        transform: rotate(15deg);
+        right: 0px;
+        top: 10px;
+        z-index: 1;
       }
     }
-  }
 
-  .master-book-list-container{
-    display: flex;
-    height: 636px;
-    width: 900px;
-    background-color: $wild-sand;
-    flex-flow: column wrap;
-    justify-content: space-around;
-    align-items: center;
-    margin-top: 36px;
+    .book-title {
+      height: 40px;
+      color: $black;
+      font-size: 32px;
+      font-weight: 700;
+      line-height: 38px;
+      align-self: center;
+      grid-area: titulo;
+      margin-left: 28px;
+    }
   
-    .book-container{
+    .book-genre {
+      height: 40px;
+      color: $gray;
+      font-size: 24px;
+      font-weight: 700;
+      line-height: 28px;
+      align-self: center;
+      margin-top: 8px;
+      grid-area: genero;
+      margin-left: 20px;
+    }
+
+    .container-text {
+      grid-area: text;
       display: flex;
-      height: 300px;
-      width: 198px;
-      background-color: $white;
-      box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
       flex-direction: column;
-      justify-content: center;
-  
-      .cover-page-list{
-        height: 200px;
-        width: 142px;
-        align-self: center;
+      margin-left: 28px;
+      
+      .container-book {
+        display: flex;
+        width: 540px;
+        margin-bottom: 36px;
+        border-bottom: 4px solid $earls-green;
       }
-  
-      .book-list-detail{
+
+      .container-detail {
         display: flex;
         flex-direction: column;
-        margin-top: 12px;
-        padding-left: 28px;
-        
-  
-        .book-list-title{
+      
+        .title {
           color: $black;
-          font-size: 16px;
+          font-size: 20px;
           font-weight: 700;
-          line-height: 18px;
-          margin-bottom: 8px;
+          line-height: 24px;
         }
-  
-        .book-list-autor{
+        
+        .detail {
           color: $gray;
-          font-size: 14px;
-          line-height: 18px;
+          font-size: 20px;
+          line-height: 24px;
+          margin-bottom: 36px;
         }
       }
     }
-
-    .book-container:hover{
-      transition: width 2s, height 2s, transform 1s;
-      transform: scaleX(1.1) scaleY(1.1);
-    }
   }
-  .cover-text-container{
-    display: flex;
-    flex-direction: row;
-    height: 416px;
-    width: 900px;
-    background-color: $white;
-    box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
-  
-    .cover-page{
-      display: flex;
-      height: 368px;
-      width: 261px;
-      margin: 24px 52px 24px 24px;
+
+  @media only screen and (max-width: $tablet){
+    height: 768px;
+    width: 1024px;
```

Comment:
$tablet is 1024px, you can use that variable here or even better: Define the rule `max-width` with that value (don't forget the width: 100%)